src/material/core/gestures/gesture-config.ts

```diff
@@ -41,19 +43,20 @@ const noopHammerInstance: HammerInstance = {
   off: () => {},
 };
 
-/** Adjusts configuration of our gesture library, Hammer. */
+/**
+ * Adjusts configuration of our gesture library, Hammer.
+ * @deprecated No longer being used. To be removed.
+ * @breaking-change 10.0.0
+ */
 @Injectable()
 export class GestureConfig extends HammerGestureConfig {
```

Comment:
Note that we can also get rid of all of this code and the one in `gesture-annotations.ts`, but it got exported through `material/core` so it's technically public API. If we don't consider it public we can remove it now, otherwise we need to wait for v10.

Reply:
I think a few people actually provide the `GestureConfig` manually. e.g. to work around a bug: https://github.com/angular/components/issues/4595#issuecomment-416641018, or if they use it to access the `slide` or `longpress` events.

I think the impact is very low if we remove it, but I could also see waiting for it until we have worked out the migration schematic.

Reply:
There are ~18 references to `GestureConfig` in Google. I think we need need some more intelligent schematic work here. Let's discuss at the team meeting tomorrow.